.github/workflows/buildpack-integration-test.yml

```diff
@@ -27,6 +27,11 @@ on:
         type: string
         default: ''
         required: false
+      builder-url:
```

Comment:
nit: if the list of builders we support is consistent for each repository (I think it should be in most cases) as an alternative to exposing the `builder-url` such that each client repository needs to specify a list of urls, what if we make this a part of the configuration matrix below on line 131?

i.e. would
```
        type: [http, cloudevent, event]
        builder-url: ['', 'gcr.io/buildpacks/builder']
        include:
          - type: http
            builder-source: ${{ inputs.http-builder-source }}
            builder-target: ${{ inputs.http-builder-target }}
          - type: cloudevent
            builder-source: ${{ inputs.cloudevent-builder-source }}
            builder-target: ${{ inputs.cloudevent-builder-target }}
          - type: event
            builder-source: ${{ inputs.event-builder-source }}
            builder-target: ${{ inputs.event-builder-target }}
```
work?

Reply:
I like this idea. Will change this back to draft to implement and test this. Thanks!